.github/workflows/build-test.yml

```diff
@@ -0,0 +1,18 @@
+name: Node.js CI
```

Comment:
Is deploy also going to be moved to github actions or is that still going to be done in Circle?

Reply:
Uhhh eventually, maybe 😅. Some sort of company mandate since GitHub actions is cheaper for the company than CircleCi